lemur/common/utils.py

```diff
@@ -299,6 +299,21 @@ def validate_conf(app, required_vars):
                 "Required variable '{var}' is not set in Lemur's conf.".format(var=var)
             )
 
+def check_validation(validation):
+    """
+    Checks that the given validation string compiles successfully.
+
+    :param validation:
+    :return str: The validation pattern, if compilation succeeds
+    """
+    
+    try:
+        compiled = re.compile(validation)
+    except re.error as e:
+        raise InvalidConfiguration(f"Validation {validation} couldn't compile. Reason: {e}")
+
+    return compiled.pattern
```

Comment:
Pushed latest commit! I will note that the `.pattern` returned by the check does escape certain characters - I'm not sure if that would affect Angular? We could return the raw `validation` instead of `compiled.pattern` if need be. Cheers! 🎉

Reply:
I checked a few of them in the UI with your changes and they look fine to me - SNS topic, Slack webook, and Vault URL/file all look likely to break if anything too weird happened in terms of escaping (but they're fine as far as I see). If there's any others you think are likely to break I can try them, but otherwise I think it seems to be fine.